python/mlcroissant/mlcroissant/_src/tests/graphs/0.8/metadata_missing_property_name/metadata.json

```diff
@@ -18,11 +20,15 @@
     "extract": "ml:extract",
     "field": "ml:field",
     "fileProperty": "ml:fileProperty",
+    "fileObject": "cr:fileObject",
```

Comment:
These datasets are 0.8 datasets, so they shouldn't have `cr` (introduced in 1.0)

Reply:
Manually diff'd these to only add `sdLicense`.